packages/api/lambdas/sf-scheduler.js

```diff
@@ -47,7 +47,6 @@ async function handleScheduleEvent(event) {
 
   const message = buildQueueMessageFromTemplate({
     messageTemplate,
-    queueUrl,
```

Comment:
This is mostly for my curiosity, do we not need to add a statement about removing this in the changelog?

Reply:
Generally speaking we use the CHANGELOG both to keep track of changes for own purposes. But also critically to notify users about user-facing changes. 

This change in particular is an internal of Cumulus, so not user facing. There are a number of those type of changes in this PR, but I'm okay with noting this one for posterity

Reply:
Oh. I take it back. `buildQueueMessageFromTemplate` is a function that is exported from a public package, so users **could** be using it and it is user facing. My bad.